tests/Moq.Tests/MockRepositoryFixture.cs

```diff
@@ -52,6 +52,44 @@ public void ShouldVerifyAll()
 			}
 		}
 
+		[Fact]
+		public void ShouldVerifyNoOtherCalls()
+		{
+			var repository = new MockRepository(MockBehavior.Default);
+			var mock = repository.Create<IFoo>();
+
+			mock.Setup(foo => foo.Do());
```

Comment:
Please remove this call to `Setup`. (`VerifyNoOtherCalls` only looks at actual invocations, it doesn't care about setups at all, so the setup is not necessary for this test.)